### ISSUE TITLE
 [JavaScript(export="false", name="")] -- generates invalid JS

### ISSUE THREAD

**Issue by bschmidtke** [RandoriAS/randori-compiler#46]
I created a class like so:

package {

```
    [JavaScript(export="false", name="")]
    public class foo
    {

        public static function bar():void
        {

        }
    }
```

}

The generated JS code referencing this method looks like the following:

.bar();

Should we get a error about this?

**Comment by teotigraphix** [comment]
The problem is we are half way into getting real Annotation support that would give a warning about this issue because the point of typed metadata is knowing when something like this is not fulfilled.

I will wait until Mike and Roland figure out what to do with the builtin.swc before I just start adding warnings for annotations. :)

**Comment by teotigraphix** [comment]
Also, by no means am I saying this does **not** need a warning, I just don't want to add code into the compiler when I know its going to be taken care of on a higher level.

There is one other thing to think about to, is this just a waring/error or should we make it still compile by using the the qualified name of the class instead. Mike, Roland?

**Comment by labriola** [comment]
No, regardless of where it is handled, I think this is warning. If a name is passed, we should use it. We can warn that its perhaps a dumb name :) but I think its a warning when we have annotations. Other than that the compiler is doing what's expected

**Comment by teotigraphix** [comment]
I have a test for this if/when we want to change it, but for now if the name="", that is exactly what the compiler produces.
